Code/GraphMol/FileParsers/PDBWriter.cpp

```diff
@@ -153,6 +138,36 @@ std::string GetPDBAtomLine(const Atom *atom, const Conformer *conf,
   return ss.str();
 }
 
+std::array<char, 2> GetDefaultAtomNumber(const Atom* atom, std::map<unsigned int, unsigned int> &elem) {
```

Comment:
I think this would be better if you used either an `std::pair<char>` (a more natural container for something which is only two long) or an `std::string` (a more natural container for something which is inherently a string).

Reply:
I changed it to be a std::string. That is also more convenient to work with in terms of output printing.